rm/rm-server/src/test/java/functionaltests/nodesource/TestSSHInfrastructureV2.java

```diff
@@ -153,24 +152,17 @@ public Command createCommand(String command) {
                 }
                 rebuiltCommand.trimToSize();
 
-                EnumSet<ProcessShellFactory.TtyOptions> ttyOptions;
                 if (OsUtils.isUNIX()) {
-                    ttyOptions = EnumSet.of(ProcessShellFactory.TtyOptions.ONlCr);
+                    return new ProcessShellFactory(new String[] { "/bin/sh", "-c",
+                                                                  rebuiltCommand.toString() }).create();
                 } else {
-                    ttyOptions = EnumSet.of(ProcessShellFactory.TtyOptions.Echo,
-                                            ProcessShellFactory.TtyOptions.ICrNl,
-                                            ProcessShellFactory.TtyOptions.ONlCr);
-                }
-
-                if (OsUtils.isUNIX()) {
-                    return new ProcessShellFactory(new String[] { "/bin/sh", "-c", rebuiltCommand.toString() },
-                                                   ttyOptions).create();
-                } else {
-                    return new ProcessShellFactory(new String[] { "cmd.exe", "/C", rebuiltCommand.toString() },
-                                                   ttyOptions).create();
+                    return new ProcessShellFactory(new String[] { "cmd.exe", "/C",
+                                                                  rebuiltCommand.toString() }).create();
```

Comment:
So the options that are by default are the ones we need?

Reply:
Good catch, the underlying library mechanism now integrate the ttyOptions logic itself 
 (detect the cariage return character based on the OS), we were previously doing this ourselves, now it's done by the library.